docker-compose.yaml

```diff
@@ -18,6 +18,17 @@ services:
       - 8888:8000
     depends_on:
       - db
+  init:
+    build: .
+    command: /bin/sh ./init/provision.sh
```

Comment:
I apologize I didn't see the command here. 
Reference: https://github.com/civictechindex/CTI-website-backend/pull/14/files#r494713459